cmd/run.go

```diff
@@ -510,7 +522,7 @@ func executeRun(runFilePath string, apps []runfileconfig.App) (bool, error) {
 	}
 
 	// Stop daprd and app processes for each runState.
-	exitWithError, closeError := gracefullyShutdownAppsAndCloseResources(runStates, apps)
+	_, closeError := gracefullyShutdownAppsAndCloseResources(runStates, apps)
```

Comment:
nit: remove 1st return param from the method `gracefullyShutdownAppsAndCloseResources` ?

Reply:
Will change in #1170

Reply:
done